src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -2168,4 +2179,14 @@ class VariantSampleMatrix[RPK, RK, T >: Null](val hc: HailContext, val metadata:
     iterations: Int = 10000): KeyTable = {
     Skat(this, variantKeys, singleKey,  weightExpr, y, x, covariates, logistic, maxSize, accuracy, iterations)
   }
+
+  def sampleQC(root: String = "sa.qc"): VariantSampleMatrix[RPK, RK, T] = {
+    requireRowKeyVariant("sample_qc")
```

Comment:
We aren't consistent on using the precise method name `sample_qc` or an informal version like `sample qc`. I prefer the former because it's unambiguous and simplifies searching the docs. If you agree, add underscores to `de novo` and others above.

Reply:
These strings should match the Python name.  It might be a holdover from a before we abandoned the Scala interface and we weren't sure what to write.  I think PR is complete and this should get fixed separately.